### DISCUSSION TITLE
Raspberry Pi 3B could work?

### DISCUSSION THREAD

**Discussion by ateodoro** [frntc/RAD#7] [Q&A]
Hi,
I have a Raspberry Pi 3B, not 3B+.
Can I use it to make a RAD Espansion Unit?
Ciao
Alberto

**Comment by frntc** [comment]
I haven't tried and not officially supported. You might be lucky and try tweaking the overclocking and timing values, but I recommend to get another RPi (3A+ seem to be available atm)

**Reply by ateodoro** [reply]
Many thanks.
Please, can you say me where I can find the config.txt ti read the overclock values, so I could evaluate to risk or not.
Ciao
Alberto

**Reply by frntc** [reply]
the clock settings are in SD:config.txt -- you can try to keep the ARM-clock at 1.4GHz or 1.45GHz and adjust core_freq and sdram_freq (maybe go as high as your 3B boots) and then use the built-in timing adjustment (given that it boots to the RAD menu). But as said, no guarantees, I never tried.

**Comment by COREi64** [comment]
The 3B works flawlessly without tweaking with the default software distribution in my experience.

**Comment by frntc** [comment]
> Hi, I have a Raspberry Pi 3B, not 3B+. Can I use it to make a RAD Espansion Unit? Ciao Alberto

There are two revisions of a RPi 3B, if you're lucky and got a new one then it might work (see https://www.raspberrypi.com/app/uploads/2019/01/RPI-PCN-Pi3B-001.pdf). Check the SoC for "BCM2837B0" (it has a metal heat spreader, as opposed to the old one)

**Reply by COREi64** [reply]
I actually have TWO 3B's.  I have one that I purchased shortly after they came to market, so it would be a very early one.  I also have a 3B that I was fortunate enough to buy in the past month.  Both work without issue.

That having been said, thank-you for your insight, I guess it's one of those... try it an see if it works arrangements eh! =)
